components/Registration/Mitra/index.vue

```diff
@@ -239,7 +239,27 @@ export default {
       const isCompanyValidated = this.form.company === '' || this.form.company === null
       const mailFormat = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/
       const isEmailValidated = this.form.email === '' || this.form.email === null || !mailFormat.test(this.form.email)
-      const isPasswordValidated = this.form.password.length < 6 && this.form.password !== this.form.password_confirm
+      const level = this.levelPassword
+      let isPasswordValidated = false
+      /**
+       * This conditional is used to pass validation password when the minimum password level is medium
+       */
+      switch (level) {
+        case 'default':
+          isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || true
+          break
+        case 'low':
+          isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || true
+          break
+        case 'medium':
+          isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || false
+          break
+        case 'strong':
+          isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || false
+          break
+        default:
+          isPasswordValidated = false
+      }
```

Comment:
#### Minor

If you have the same logic in some cases, you can simplify the code like this

```js
switch (level) {
  case 'default':
  case 'low':
    isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || true
    break
  case 'medium':
  case 'strong':
    isPasswordValidated = (this.form.password.length < 6 && this.form.password !== this.form.password_confirm) || false
    break
  default:
    isPasswordValidated = false
}
```

Reply:
it's a nice info kak @bangunbagustapa, thanks kak